### ISSUE TITLE
Add a contributing.md file

### ISSUE THREAD

**Issue by cowlicks** [EFForg/privacybadger#1195]
To make Privacy Badger more welcoming to new contributors, it would be nice to have a more detailed explanation of the development process. Some of this can be copied from the README, it is probably fine for it to be duplicated.

Some new stuff that should be included:
- [x] How to install in Firefox with web-ext and run qunit tests (fixed in #1437)
- [x] How to address "site-bug" issues, different kind of issues (partially addressed in #1451, see [this comment](https://github.com/EFForg/privacybadger/pull/1451#discussion_r123865617) for possible further work). 
- [x] The current instructions for selenium tests are broken (add ff instructons after #1168 is in) (fixed in #1437)
- [ ] Explain how dependencies are managed
- [ ] Mention debug flag
- [ ] Mention how 400 errors in the console are normal and from checking for DNT policies 
- [ ] A high level description of architecture so new contributors can orient themselves to the codebase more quickly
- [x] Note how translations are handled. Related to #1280

Anything else? This is a lot of stuff, so please feel free to submit a partial PR, more details can always be added later.

**Comment by cowlicks** [comment]
Added translations note

**Comment by cowlicks** [comment]
Here is a legacy PB's [contributing.md](https://github.com/EFForg/privacybadgerfirefox-legacy/blob/master/CONTRIBUTING.md).

**Comment by cowlicks** [comment]
Also we already a [CONTIRBUTION-GUIDELINES.md](https://github.com/EFForg/privacybadger/blob/master/doc/CONTRIBUTION-GUIDELINES.md).

**Comment by cooperq** [comment]
+1!! You should start by making a PR which includes yourself and everyone listed here: https://github.com/EFForg/privacybadger/graphs/contributors
https://github.com/EFForg/privacybadgerfirefox/graphs/contributors

**Comment by cooperq** [comment]
Oh, I reread the issue and see that you meant something different which is also a great idea! I was trying to say we should also have a list of contributors (to acknowledge peoples' hard work)

**Comment by ghostwords** [comment]
WIP "how to fix broken sites" doc here: https://github.com/EFForg/privacybadger/pull/1451

**Comment by cowlicks** [comment]
When #1221 and #1430 are merged we should add a note to the `CONTRIBUTING.md` about how to use them.

**Comment by ghostwords** [comment]
I think we're in a good place after #1468 and related PRs. Let's close this and open followup issues, one for each potential improvement?

**Comment by ghostwords** [comment]
Resolving. We can open new documentation issues as we realize the need for more/better docs.
